src/components/modals/TransactionModal/TransactionModal.tsx

```diff
@@ -0,0 +1,209 @@
+      .map(([name, id]) => {
+        if (id === txId.toString()) return name;
+      })
+      .filter((n) => !!n);
+  }
+
+  return (
+    <>
+      <div
+        className="modal-container"
+        style={isMobile ? { padding: 'none' } : {}}
+      >
+        {/**modal header */}
+        <div
+          className="flex flex-column card"
+          style={
+            isMobile
+              ? {
+                  width: '95%',
+                  maxWidth: 'none',
+                  height: '95%',
+                  padding: 0,
+                  boxSizing: 'border-box',
+                  gap: '1em',
+                }
+              : {
+                  width: 400,
+                  minHeight: 200,
+                  padding: 0,
+                  paddingBottom: 15,
+                  gap: '1em',
+                }
+          }
+        >
+          <div
+            className="flex flex-column flex-space-between"
+            style={{
+              borderBottom: '2px rgb(0,0,0,.2) solid',
+              padding: 15,
+              boxSizing: 'border-box',
+              marginTop: 5,
+              gap: '.5em',
+              position: 'relative',
+            }}
+          >
+            <span className="text-medium white bold">
+              {interactionType}&nbsp;:&nbsp;{state?.state.ticker}
+            </span>
+            <span className="flex faded text" style={{ alignItems: 'center' }}>
+              Contract ID:&nbsp;
+              <CopyTextButton
+                copyText={contractId ? contractId.toString() : ''}
+                displayText={`${contractId!
+                  .toString()
+                  .slice(0, isMobile ? 6 : 15)}...${contractId!
+                  .toString()
+                  .slice(isMobile ? -6 : -15)}`}
+                size={25}
+                wrapperStyle={{
+                  fontStyle: 'bold',
+                  justifyContent: 'center',
+                  alignItems: 'center',
+                }}
+                position="relative"
+              />
+            </span>
+            <button className="modal-close-button" onClick={() => showModal()}>
+              <CloseIcon
+                width="20px"
+                height={'20px'}
+                fill="var(--text-white)"
+              />
+            </button>
+          </div>
+          {/** modal body - condition render edit or transfer */}
+          {interactionType === ANT_INTERACTION_TYPES.TRANSFER ? (
+            <div
+              className="flex flex-column"
+              style={{
+                width: '90%',
+                height: '100%',
+                gap: '1em',
+              }}
+            >
+              {/*                        todo: show 'transfer between accounts' option     
+                         {addreses ? <span className="flex flex-column white text">{Object.entries(addreses).map(([address, name])=> <span>{`${name} : ${address}`}</span>)}</span> : <></>}
+
+                         */}
```

Comment:
remove this and just leave the todo comment

Reply:
removed